controllers/cluster/cluster_controller_test.go

```diff
@@ -367,7 +369,7 @@ func proberShouldBePresent(g *WithT, reconciler *Reconciler, cluster *gardenerv1
 	g.Eventually(func() int { return len(reconciler.ProberMgr.GetAllProbers()) }, 10*time.Second, 1*time.Second).Should(Equal(1))
```

Comment:
The last parameter should ideally be named as `expectedKCMNodeMonitorGraceDuration`, right?